rubin_changelog/rst.py

```diff
@@ -0,0 +1,202 @@
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import logging
+from typing import List
+
+from rstcloth import RstCloth
+
+from rstcloth.rstcloth import Table
+from sortedcontainers import SortedDict, SortedList
+from .tag import Tag
+from dateutil.parser import parse
+
+log = logging.getLogger(__name__)
+
+
+class Writer:
+    """Create RST output files"""
```

Comment:
The other way to do this would be to fill in a template (e.g. with jinja2).  But this is fine.